app/routes/events/view/speakers/create.js

```diff
@@ -0,0 +1,34 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  titleToken() {
+    return this.get('l10n').t('Create session');
+  },
+  async model() {
+    const eventDetails = this.modelFor('events.view');
+    return {
+      event : eventDetails,
+      form  : await eventDetails.query('customForms', {
+        'page[size]' : 50,
+        sort         : 'id'
+      }),
+      session: await this.get('store').createRecord('session', {
+        event   : eventDetails,
+        creator : this.get('authManager.currentUser')
+      }),
+      sessions : eventDetails.get('store').findAll('session'),
+      speaker  : await this.get('store').createRecord('speaker', {
+        event : eventDetails,
+        user  : this.get('authManager.currentUser')
+      }),
+      tracks       : await eventDetails.query('tracks', {}),
+      sessionTypes : await eventDetails.query('sessionTypes', {})
+    };
+  },
+  resetController(controller) {
+    this._super(...arguments);
+    const model = controller.get('model');
+    model.speaker.unloadRecord();
```

Comment:
@ritikamotwani This will remove the record from the store as soon as we transition to another route. We should change this as done in this PR. https://github.com/fossasia/open-event-frontend/pull/1318, this PR does not cover this file so we can update this here itself.